.terraform-version

```diff
@@ -1 +1 @@
-0.13.5
+0.13.7
```

Comment:
If you change this value also change it in the `.circleci/config.yml` so that they are consistent.